kfserving/kfserving-crds/overlays/application/kustomization.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
 bases:
 - ../../base
-resources:
-- application.yaml
 commonLabels:
-  app.kubernetes.io/name: kfserving-crds
-  app.kubernetes.io/instance: kfserving-crds-v1.0.0
-  app.kubernetes.io/managed-by: kfctl
   app.kubernetes.io/component: kfserving-crds
+  app.kubernetes.io/instance: kfserving-crds-0.2.2
+  app.kubernetes.io/managed-by: kfctl
+  app.kubernetes.io/name: kfserving-crds
   app.kubernetes.io/part-of: kubeflow
-  app.kubernetes.io/version: v1.0.0
+  app.kubernetes.io/version: 0.2.2
```

Comment:
same here?

Reply:
Yup. This should be the KFServing version.